middleware.js

```diff
@@ -0,0 +1,24 @@
+import { userData } from './data/index.js';
+
+const middleware = {
```

Comment:
Not sure if these functions can be considered as middleware. These functions did not impact the current transaction or the next middleware. These functions just simply check if the user is logged in or is a gym owner. 
Should belongs to the helper method. If you want to add a function in middleware, you can make changes in app.js. There is one template, which can be used for logging feature.

Reply:
I agree with Xinxuan. Middleware should be something that's happen before it hit the routes, and these two functions checkIfLoggedIn and checkIfGymOwner, even though with similar results, resides within the routes thus is a helper/validation method. 

But we can reduce the duplicate codes in the middleware inside app.js by directly calling these two methods?